### ISSUE TITLE
NTR: Liver biopsy: acetyl-CoA carboxylase low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9391]
New term request Liver biopsy: acetyl-CoA carboxylase low (elthat_151028073155):   UMLS: 3279541

**Comment by pnrobinson** [comment]
@MickeySegal
Is that https://omim.org/entry/613933
Is this a specifically liver phenotype?

**Comment by MickeySegal** [comment]
It is https://www.omim.org/entry/613933.  The phenotype does seem wider, including muscle, so it is fine to not make this liver-specific.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000430
- label: Reduced tissue acetyl-CoA carboxylase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:34552920
- def: Activity of acetyl-CoA carboxylase in the tissues below the lower limit of normal. The activity can be measured in multiple tissues including liver.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9391
- synonyms: Reduced hepatic acetyl-CoA carboxylase activity(NONE)
